<ui version="4.0" >
 <class>PeakFitDialog</class>
 <widget class="QDialog" name="PeakFitDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>391</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>642</width>
    <height>391</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>642</width>
    <height>391</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Fit peaks</string>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>350</y>
     <width>621</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnFit" >
      <property name="text" >
       <string>Fit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose" >
      <property name="text" >
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>329</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>Fit settings</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget" >
    <property name="geometry" >
     <rect>
      <x>9</x>
      <y>19</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="midLineWidth" >
        <number>0</number>
       </property>
       <property name="text" >
        <string>Fitting function</string>
       </property>
       <property name="buddy" >
        <cstring></cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFunction" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <item>
        <property name="text" >
         <string>Gaussian</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Lorentzian</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblExpression" >
       <property name="text" >
        <string>Expression</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leExpression" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableParams" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <row>
        <property name="text" >
         <string>New Row</string>
        </property>
       </row>
       <column>
        <property name="text" >
         <string>Parameter</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Fixed/Value</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>10</y>
     <width>301</width>
     <height>329</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>Input / Output</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2" >
    <property name="geometry" >
     <rect>
      <x>9</x>
      <y>19</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Input workspace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbInWorkspace" />
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <property name="sizeConstraint" >
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Spectrum number</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="leSpectrum" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Output workspace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leOutWorkspace" />
     </item>
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Parameter table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leParamTable" />
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbFunction</tabstop>
  <tabstop>leExpression</tabstop>
  <tabstop>tableParams</tabstop>
  <tabstop>cbInWorkspace</tabstop>
  <tabstop>leSpectrum</tabstop>
  <tabstop>leOutWorkspace</tabstop>
  <tabstop>leParamTable</tabstop>
  <tabstop>btnFit</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
